gke-deploy/core/image/image.go

```diff
@@ -47,31 +40,32 @@ func ParseImages(ctx context.Context, images []string) ([]Image, error) {
 			return nil, fmt.Errorf("duplicate image name: %q", imName)
 		}
 		exists[imName] = true
-		ims = append(ims, im)
+		refs = append(refs, ref)
 	}
 
-	return ims, nil
+	return refs, nil
 }
 
-func parseImage(ctx context.Context, image string) (Image, error) {
+func parseImage(image string) (name.Reference, error) {
 	im, err := name.ParseReference(image)
 	if err != nil {
 		return nil, fmt.Errorf("image is invalid: %q", image)
 	}
 	return im, nil
 }
 
-// GetNameFromString gets an image's name, given a string representation of the image.
-func GetNameFromString(ctx context.Context, image string) (string, error) {
-	im, err := parseImage(ctx, image)
+// ParseImageReference gets an image's name, given a string representation of the image.
+// e.g., given "gcr.io/my-project/my-app:1.0.0", returns "gcr.io/project/my-app"
+func ParseImageReference(image string) (string, error) {
```

Comment:
`ParseImageReference` isn't very descriptive here either. It parses, but then it also returns only part of what it parsed out.